spec/support/shared_examples/participant_actions_defer_support.rb

```diff
@@ -11,6 +11,7 @@
   it "creates a deferred state and makes the profile deferred" do
     expect { described_class.call(params: participant_params) }.to change { ParticipantProfileState.count }.by(1)
     expect(user_profile.participant_profile_state.deferred?)
+    expect(user_profile.training_status_deferred?)
```

Comment:
that's not going to do anything

`expect(user_profile).to be_training_status_deferred`